martify/hyresgast
# Client setup

Added gulp task runner to client

## Prerequisites/How to test
1. run `gulp serve-dev`
2. edit a file. runner should re-run
3. run `gulp serve-prod`
4. look att files in dist folder

*Not included*: I have not tested live reload and i believe gulp serve-dev needs to be restarted if files are added.

TODO - Make gulp handle:
1. Live reload
2. Fonts
3. Images
